jobs/FunctionTest/prepare_manifest.sh

```diff
@@ -118,6 +129,9 @@ prepareDeps(){
   preparePackages
   dlTftpFiles
   dlHttpFiles
```

Comment:
talking with @sunnyqianzhang ,  the dlHttpFiles can moved to else and no harm to put it here